isobus/src/can_control_function.cpp

```diff
@@ -46,6 +46,11 @@ namespace isobus
 		return controlFunctionNAME;
 	}
 
+	void ControlFunction::set_NAME(const NAME &newNAME)
+	{
+		controlFunctionNAME = newNAME;
```

Comment:
What's the intent of this setter? IDK if we would want it exposed to everything, because if it's called on an internal control function it will not restart address claiming or anything....

Reply:
I used this to alter the NAME of the CF here, right after it's initialization:
https://github.com/martonmiklos/AgIsoVirtualTerminal/blob/support_setting_vt_number/src/ServerMainComponent.cpp#L51

What do you think if I move all the settings handling in AgIsoVT to a separate class what I can initialize before the creation of the ServerMainComponent handle the CF Name change before the CF initialization. This way we can get rid from this change in the stack.

Reply:
I think that would be fine. Changing the internal control function's NAME properly would be a bit complicated, as we'd have to set the address claim state back to none, and mark it as invalid/no address in the network manager also for it to really work after initialization

Reply:
I sorted out in AgIsoVT the settings loading, so I could remove this method. 
Are you comfortable with the rest of the changes in this PR?